android/src/main/java/com/revenuecat/purchases/react/RNPurchasesModule.java

```diff
@@ -463,6 +459,16 @@ public void onReceived(Boolean result) {
     }
 
     @ReactMethod
+    public void syncAmazonPurchase(String productID, String receiptID,
+                                               String amazonUserID, String isoCurrencyCode,
+                                               Double price, final Promise promise) {
+      Purchases.getSharedInstance().syncAmazonPurchase(productID, receiptID,
+        amazonUserID, isoCurrencyCode, price);
+      promise.resolve(null);
+    }
+
+    @ReactMethod
+    @Deprecated // Use syncAmazonPurchase instead
```

Comment:
we could make `syncObserverModeAmazonPurchase` call this method so it's not duplicated?